content/docs/01-introduction/palette-freemium.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Try Palette for Free?"
+metaTitle: "About Free Tier"
+metaDescription: "Palette Free Tier, Freemium"
+icon: ""
+hideToC: true
+fullWidth: false
+---
+
+# Trying Palette for Free
+
+**Palette Free Tier** encourages new users to explore Palette, without having to make a purchase decision, before they are ready. With unlimited kilo-Core-hour usage the first month, there is full access to Spectro Cloud Palette to create, deploy, and manage Cluster Profiles and Projects. 
+
+The second month and every month thereafter, the customer is granted a complimentary 25 kilo-Core-hour (kCh) usage to use freely to manage their Kubernetes clusters with Palette. 
+
+There is the choice to continue to stay in the Free Tier indefinitely, so long as you stay under the 25 kCh consumption. Go over the usage limit of 25 kCh, and the clusters remain visible, but convert to Read-Only. Enter a payment and all usage and capabilities are restored.
+
+
+# Kilo-Core-hour Calculations
+
+Usage is calculated as a concept of kilo-Core-hour or kCh. This is a measurement of the number of CPU Cores multiplied by Time. For instance, for a 4-node cluster which has 16 CPU cores each, one day of management will equate to 4 x 16 CPU cores x 24 hours = 1.54 kCh.
+
+
```

Comment:
Palette sends alert messages to users when 25 kCh is crossed
and if more than 5 active clusters are launched further deployments will be restricted.
We can consider including these points as well after confirming. @truegilb plz confirm.